src/uucore/src/lib/lib.rs

```diff
@@ -146,15 +146,16 @@ pub trait Args: Iterator<Item = OsString> + Sized {
                 InvalidEncodingHandling::Ignore => s.is_ok(),
                 _ => true,
             })
-            .map(|s| match s.is_ok() {
-                true => s.unwrap(),
-                false => s.unwrap_err(),
+            .map(|s| match s {
+                Ok(v) => v,
+                Err(e) => e,
             })
             .collect();
 
-        match full_conversion {
-            true => ConversionResult::Complete(result_vector),
-            false => ConversionResult::Lossy(result_vector),
+        if full_conversion {
```

Comment:
I love this one :)

Reply:
That was definitely an interesting blooper :)